independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1058 @@
+                      },
+                      "isEffectiveIdentityRequired": {
+                        "type": "boolean",
+                        "description": "Whether the dataset requires an effective identity. This indicates that you must send an effective identity using the GenerateToken API."
+                      },
+                      "isEffectiveIdentityRolesRequired": {
+                        "type": "boolean",
+                        "description": "Whether RLS is defined inside the PBIX file. This indicates that you must specify a role."
+                      },
+                      "isOnPremGatewayRequired": {
+                        "type": "boolean",
+                        "description": "Dataset requires an On-premises Data Gateway"
+                      }
+                    }
+                  },
+                  "description": "value"
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			      "x-ms-summary" : "Enter filter criteria",
+			      "description" : "Filters the results, based on a boolean condition"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			      "x-ms-summary" : "Number of records to return",
+            "description": "Use the top parameter to set the first n results"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": false,
+            "type": "string",
+			      "x-ms-summary" : "Number of records to skip",
+            "description": "Use the top parameter to skip the first x amount of groups that are being returned."
+          }
+        ],
+        "operationId": "DatasetsGetDatasetsAsAdmin",
+        "summary": "Get Datasets As Admin",
+        "description": "Returns a list of datasets for the organization"
+      }
+    },
+    "/v1.0/myorg/admin/groups/{groupId}/datasets": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "name"
+                      },
+                      "addRowsAPIEnabled": {
+                        "type": "boolean",
+                        "description": "addRowsAPIEnabled"

```

Comment:
@paulculmsee Can you please provide valid description values, there are few other description properties need valid values.